### PR TITLE
Update Typescript and typings.

### PR DESCRIPTION
**Pull request by jtlan**
Typescript --> 1.4.1
d3.d.ts --> newest

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -223,7 +223,7 @@ module.exports = function(grunt) {
           testname: 'SVGTypewriter Sauce Unit Tests',
           browsers: [{
             browserName: "firefox",
-            version: "30" 
+            version: "30"
```
Comment (aicioara) [inline Gruntfile.js]: We may want to not specify the version number for both FF and chrome, such that it is always the latest on SauceLabs

File: test/testSuite/wrapperTestSuite.ts [hunk 2]
```diff
@@ -48,7 +48,6 @@ describe("Wrapper Test Suite", () => {
 
     it("does not wrap", () => {
       var dimensions = measurer.measure(token);
-      debugger;
```
Comment (aicioara) [inline test/testSuite/wrapperTestSuite.ts]: We may want to enforce this in a tslint?

**Comment by aicioara** [comment]
I was assuming you're going to remove 
```haml
addons:
  sauce_connect: true
```
from `.travis.yml`

**Comment by aicioara** [comment]
All comments are outside of the scope of the PR, so feel free to merge this and open different PR later for the flags.

**Comment by jtlan** [comment]
Fixing the sauce stuff, since that might catch bugs.

File: test/testSuite/wrapperTestSuite.ts (hunk 2 above)
Comment (jtlan) [inline test/testSuite/wrapperTestSuite.ts]: --> separate PR later.

**Pull request merged**